drivers/xen/gnttab.c

```diff
@@ -341,12 +341,12 @@ static int gnttab_init(void)
 		gnttab.gref_list[0] = gref;
 	}
 
-	for (i = 0; i < CONFIG_NR_GRANT_FRAMES; i++) {
+	for (i = CONFIG_NR_GRANT_FRAMES; i; i--) {
```

Comment:
would it be better to initialise `i` with `CONFIG_NR_GRANT_FRAMES  - 1`, instead of using `i - 1` twice and using `i >=0` as condition?

Reply:
Yeah, great point, it is also decreasing amount of lines touched by commit, done. Thanks!